apps/mobile/src/components/splash-screen-guard/use-auth-state.tsx

```diff
@@ -60,9 +60,17 @@ export function useAuthState({
     if (securityLevelPreference === 'secure') {
       setAnimationFinished(false);
       setAuthState('started');
-      userLeavesApp(+new Date());
+      // add latest active timestamp only if the app was actually unlocked
+      if (authState === 'passed-on-first' || authState === 'passed-afterwards')
```

Comment:
How about replacing the comment with a const? 

```suggestion
      const appUnlocked = authState === 'passed-on-first' || authState === 'passed-afterwards';
      if (appUnlocked)
```